### PR TITLE
docs: readme update

### PR DESCRIPTION
**Pull request by m1ga**
* Atom -> Pulsar
* Appcelerator -> :cloud: 
* legal text update
* add ppm for install and git repo method

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
+Use `ppm` (Pulsar package manager) or `apm` (Atom package manager):
+
+```
+ppm install pulsar-titanium
+```
+
+You can also use the repo URL to install the latest repo version:
 ```
-apm install appcelerator-titanium
+ppm install https://github.com/tidev/pulsar-titanium
```
Comment (ewanharris) [inline README.md:26]: I think they're leaning towards `pulsar --package` now over `ppm` (and I've had issues with it on Windows)
```suggestion
pulsar --package install https://github.com/tidev/pulsar-titanium
```

File: README.md [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
 
-Preferences/Settings > Install > Search for `appcelerator-titanium`.
+Preferences/Settings > Install > Search for `pulsar-titanium`.
 
 ### Using the CLI:
 
+Use `ppm` (Pulsar package manager) or `apm` (Atom package manager):
+
+```
+ppm install pulsar-titanium
```
Comment (ewanharris) [inline README.md:21]:
```suggestion
pulsar --package install pulsar-titanium
```

File: README.md [hunk 3]
```diff
@@ -1,23 +1,29 @@
...
-<img src="https://user-images.githubusercontent.com/10667698/42400977-8e552cde-8141-11e8-8696-1380eb9dd581.png" width="900px" alt="Appcelerator Titanium for Atom screenshot" />
+<img src="https://user-images.githubusercontent.com/10667698/42400977-8e552cde-8141-11e8-8696-1380eb9dd581.png" width="900px" alt="Titanium SDK for Pulsar screenshot" />
 
 ## Getting Started
 
-* Atom 1.21 (or later) is required.
-* Some features are implemented as providers for services in the [Atom IDE](https://ide.atom.io) package. Install `atom-ide-ui` package.
-* Install `appcelerator-titanium` package.
+* Pulsar is required.
+* Install `pulsar-titanium` package.
```
Comment (ewanharris) [inline README.md:10]: Package name is just `titanium`, I think we do still need some extra packages but I'll need to check what else is needed

```suggestion
* Install `titanium` package.
```

File: README.md [hunk 4]
```diff
@@ -1,23 +1,29 @@
...
 
-### Using the Atom package manager
+### Using the Pulsar package manager
 
-Preferences/Settings > Install > Search for `appcelerator-titanium`.
+Preferences/Settings > Install > Search for `pulsar-titanium`.
```
Comment (ewanharris) [inline README.md:14]: Package name is just titanium

```suggestion
Preferences/Settings > Install > Search for `titanium`.
```

File: README.md [hunk 5]
```diff
@@ -93,20 +99,20 @@ Show or hide the console which outputs debug information during build and run.
 
 #### Build
 
-The build select button executes and provides selections of the build command. Depending on the target platform selected, the drop-down will display options for the `appc run` command:
+The build select button executes and provides selections of the build command. Depending on the target platform selected, the drop-down will display options for the `ti run` command:
```
Comment (ewanharris) [inline README.md:102]:
```suggestion
The build select button executes and provides selections of the build command. Depending on the target platform selected, the drop-down will display options for the `ti build` command:
```

File: README.md [hunk 6]
```diff
@@ -93,20 +99,20 @@ Show or hide the console which outputs debug information during build and run.
...
 
 <img src="https://user-images.githubusercontent.com/2697821/33169524-ed2b85ca-d03c-11e7-881e-4eb9e301649d.png" width="109px" alt="Build command select" />
 
 * Run: Build and run the app on selected simulator/emulator or device
 * Ad-Hoc: (iOS only) build and package the app for Ad-Hoc distribution using selected code signing
 * Publish: Build and package the app for distribution through the App Store or Google Play using selected code signing or keystore credentials
-* Custom: Execute `appc run -d <project_dir>` with custom arguments
+* Custom: Execute `ti run -d <project_dir>` with custom arguments
```
Comment (ewanharris) [inline README.md:109]:
```suggestion
* Custom: Execute `ti build -d <project_dir>` with custom arguments
```

File: README.md [hunk 7]
```diff
@@ -93,20 +99,20 @@ Show or hide the console which outputs debug information during build and run.
...
 
 #### Console
 
 <img src="https://user-images.githubusercontent.com/2697821/33169526-ed531cf2-d03c-11e7-8d08-5657f691aca2.png" width="876px" alt="Build command select" />
 
-The console displays debug logging during the build and run. The log level is passed to the `appc run` comand so selecting the log level does not alter output from the currently executing command. There are options to automatically scroll the console to the latest message and to open it when a build is initiated. 
+The console displays debug logging during the build and run. The log level is passed to the `appc run` comand so selecting the log level does not alter output from the currently executing command. There are options to automatically scroll the console to the latest message and to open it when a build is initiated.
```
Comment (ewanharris) [inline README.md:115]:
```suggestion
The console displays debug logging during the build and run. The log level is passed to the `ti build` command so selecting the log level does not alter output from the currently executing command. There are options to automatically scroll the console to the latest message and to open it when a build is initiated.
```

**Review by ewanharris — CHANGES_REQUESTED** [review]
Just some comments, but this is ace thanks!

**Comment by m1ga** [comment]
thanks for the commit suggestions :+1: All valid of course!

**Review by ewanharris — APPROVED** [review]
Thanks @m1ga!

**Pull request merged**